src/styles/style-stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars--4 {
+  display: flex;
+  flex-direction: row;
+  width: 96px;
+  height: 16px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
It works only for 4 stars for now. Reuse the whole stars block, please 
![image](https://github.com/mate-academy/layout_product-cards/assets/99403198/1bac7e09-5327-46a1-98da-c9f333bc62dc)